components/brave_ads/browser/ad_notification.cc

```diff
@@ -3,7 +3,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "brave/browser/version_info.h"
```

Comment:
we cannot have dependencies on `brave/browser` in components

Reply:
removed